specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -4473,6 +4473,52 @@
             }
           }
         }
+      },
+      "put": {
+        "tags": [
+          "BillingRoleAssignments"
+        ],
+        "operationId": "EnrollmentRoleAssignments_Put",
+        "description": "Create or update a billing role assignment.",
+        "x-ms-examples": {
+          "PutEnrollmentAdministratorRoleAssignment": {
+            "$ref": "./examples/PutEnrollmentAdministratorRoleAssignment.json"
```

Comment:
Same here - please remove "Enrollment" from any naming for billing account-level APIs.